test/run-e2e-ocp.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+set -e -u -o pipefail
+
+declare PROJECT_ROOT
+PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+declare OPERATORS_NS
+OPERATORS_NS="openshift-operators"
+declare NO_INSTALL=false
+declare NO_UNINSTALL=false
+declare SHOW_USAGE=false
+
+install_obo_OCP(){
+  oc apply -f ./hack/olm/catalog-src.yaml
+  oc apply -f ./hack/olm/subscription.yaml
+}
```

Comment:
we should wait for the catalog-src to be ready before subscribing. 
We should also wait for the subscription to be complete before exiting the function.